### ISSUE TITLE
错误提示

### ISSUE THREAD

**Issue by ruinjoy** [andresayac/edge-tts-php#1]
使用示例代码提示

Fatal error: Uncaught Error: Class "Afaya\EdgeTTS\Service\EdgeTTS" not found

**Comment by andresayac** [comment]
Check This please

https://github.com/user-attachments/assets/79d7c305-12e2-48aa-8161-00d433556a0f

**Comment by ruinjoy** [comment]
`./composer.json has been created
Running composer update afaya/edge-tts
Loading composer repositories with package information
Updating dependencies
Lock file operations: 26 installs, 0 updates, 0 removals
  - Locking afaya/edge-tts (1.1.5)
  - Locking brick/math (0.12.1)
  - Locking evenement/evenement (v3.0.2)
  - Locking guzzlehttp/psr7 (2.7.0)
  - Locking psr/container (2.0.2)
  - Locking psr/http-factory (1.1.0)
  - Locking psr/http-message (2.0)
  - Locking ralouphie/getallheaders (3.0.3)
  - Locking ramsey/collection (2.0.0)
  - Locking ramsey/uuid (4.7.6)
  - Locking ratchet/pawl (v0.4.1)
  - Locking ratchet/rfc6455 (v0.3.1)
  - Locking react/cache (v1.2.0)
  - Locking react/dns (v1.13.0)
  - Locking react/event-loop (v1.5.0)
  - Locking react/promise (v3.2.0)
  - Locking react/socket (v1.16.0)
  - Locking react/stream (v1.4.0)
  - Locking symfony/console (v7.1.5)
  - Locking symfony/deprecation-contracts (v3.5.0)
  - Locking symfony/polyfill-ctype (v1.31.0)
  - Locking symfony/polyfill-intl-grapheme (v1.31.0)
  - Locking symfony/polyfill-intl-normalizer (v1.31.0)
  - Locking symfony/polyfill-mbstring (v1.31.0)
  - Locking symfony/service-contracts (v3.5.0)
  - Locking symfony/string (v7.1.5)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 26 installs, 0 updates, 0 removals
  - Installing symfony/polyfill-mbstring (v1.31.0): Extracting archive
  - Installing symfony/polyfill-intl-normalizer (v1.31.0): Extracting archive
  - Installing symfony/polyfill-intl-grapheme (v1.31.0): Extracting archive
  - Installing symfony/polyfill-ctype (v1.31.0): Extracting archive
  - Installing symfony/string (v7.1.5): Extracting archive
  - Installing symfony/deprecation-contracts (v3.5.0): Extracting archive
  - Installing psr/container (2.0.2): Extracting archive
  - Installing symfony/service-contracts (v3.5.0): Extracting archive
  - Installing symfony/console (v7.1.5): Extracting archive
  - Installing react/event-loop (v1.5.0): Extracting archive
  - Installing evenement/evenement (v3.0.2): Extracting archive
  - Installing react/stream (v1.4.0): Extracting archive
  - Installing react/promise (v3.2.0): Extracting archive
  - Installing react/cache (v1.2.0): Extracting archive
  - Installing react/dns (v1.13.0): Extracting archive
  - Installing react/socket (v1.16.0): Extracting archive
  - Installing ralouphie/getallheaders (3.0.3): Extracting archive
  - Installing psr/http-message (2.0): Extracting archive
  - Installing psr/http-factory (1.1.0): Extracting archive
  - Installing guzzlehttp/psr7 (2.7.0): Extracting archive
  - Installing ratchet/rfc6455 (v0.3.1): Extracting archive
  - Installing ratchet/pawl (v0.4.1): Extracting archive
  - Installing ramsey/collection (2.0.0): Extracting archive
  - Installing brick/math (0.12.1): Extracting archive
  - Installing ramsey/uuid (4.7.6): Extracting archive
  - Installing afaya/edge-tts (1.1.5): Extracting archive
6 package suggestions were added by new dependencies, use `composer suggest` to see details.
Generating autoload files
No security vulnerability advisories found.
Using version ^1.1 for afaya/edge-tts
`
PHP版本8.3

**Comment by ruinjoy** [comment]
[voice_5MprP.tar.gz](https://github.com/user-attachments/files/17396986/voice_5MprP.tar.gz)
文件

**Comment by ruinjoy** [comment]
在宝塔和1panel下都提示Fatal error: Uncaught Error: Class "Afaya\EdgeTTS\Service\EdgeTTS" not found in app.php on line 8

**Comment by ruinjoy** [comment]
`版本信息
PHP版本：8.3.8
已安装扩展
Core=8.3.8
date=8.3.8
libxml=8.3.8
openssl=8.3.8
pcre=8.3.8
sqlite3=8.3.8
zlib=8.3.8
ctype=8.3.8
curl=8.3.8
dom=20031129
fileinfo=8.3.8
filter=8.3.8
hash=8.3.8
iconv=8.3.8
json=8.3.8
mbstring=8.3.8
SPL=8.3.8
session=8.3.8
PDO=8.3.8
pdo_sqlite=8.3.8
standard=8.3.8
posix=8.3.8
random=8.3.8
readline=8.3.8
Reflection=8.3.8
Phar=8.3.8
SimpleXML=8.3.8
tokenizer=8.3.8
xml=8.3.8
xmlreader=8.3.8
xmlwriter=8.3.8
mysqlnd=mysqlnd 8.3.8
cgi-fcgi=8.3.8
apcu=5.1.24
exif=8.3.8
gd=8.3.8
igbinary=3.2.16
imagick=@PACKAGE_VERSION@
intl=8.3.8
mysqli=8.3.8
pdo_mysql=8.3.8
redis=6.0.2
shmop=8.3.8
sodium=8.3.8
zip=1.22.3
memcached=3.2.0
Zend OPcache=8.3.8`

**Comment by andresayac** [comment]
I have identified the error and I will correct it. I will upload the change in a few minutes. I appreciate your contribution.

**Comment by ruinjoy** [comment]
Yes, this is a case sensitivity issue. It works fine on Windows.

**Comment by andresayac** [comment]
Check
```bash
composer require afaya/edge-tts
```

https://github.com/andresayac/edge-tts-php/releases/tag/v1.2.8
